ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,70 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        target: download_tools
+
+    - name: Create OpenStack namespace
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: namespace
+        params:
+          OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"
+
+    - name: Generate olm artifacts
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack_prep
+        params:
+          OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"
+
+    - name: Deploy Openstack Operators
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack
+        params:
```

Comment:
I wonder if we want to use yaml anchors here to avoid the repetition with params: OUT:

params: &install_yamls_out
    OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"

then we have 
params: *install_yamls_out

just a thought

Reply:
Thank you, added it in the patch.

Reply:
Thank you @marios, included it in the patch.